### ISSUE TITLE
9.4.1.2/3.1 The "More apps" menu in the mobile (responsive) view has been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened. However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42229]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n1

9.4.1.2/3.1 The "More apps" menu in the mobile (responsive) view has been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened. However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing.

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41888
